.github/workflows/tag-release.yml

```diff
@@ -71,9 +71,13 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           RELEASE_NAME: ${{ steps.get_version.outputs.version }}
         run: |
+          echo "Anaconda release $RELEASE_NAME created automatically from git tag.\n\n" > release.txt
```

Comment:
We should remove this line. It's really not useful information to users that the release was created from the git tag automatically.

Just simple:

`Anaconda release $RELEASE_NAME.` should be enough.

Reply:
Yes, that makes sense.